File: AT01/C/Q02.c
#include <stdio.h>

int main(){
    float x = 0, des = 0, res = 0;
    scanf("%f", &x);

    if (x <= 600){

        des = 0.0;
        printf("%.2f\n%.2f", des,x);
        
    }else if (x > 600 && x <= 1200){

        des = x * 0.2;
        res = x - des;
        printf("%.2f\n%.2f", des,res);

    }else if (x > 1200 && x <= 2000){

        des = x * 0.25;
        res = x - des;
        printf("%.2f\n%.2f", des,res);

    }else if(x > 2000){

        des = x * 0.3;
        res = x - des;
        printf("%.2f\n%.2f", des,res);

    }
      
}

File: AT01/py/Q02.py
y = int(input())

def desconto():
    if y <= 600 :
        des = 0.0
        print(des,"\n",y)

    elif y > 600 and y <= 1200:
        des = y * 0.20
        
        print(des,"\n",y-des)

    elif y > 1200 and y <= 2000:
        des = y * 0.25
       
        print(des,"\n",y-des)

    elif y > 2000:
        des = y * 0.30
        
        print(des,"\n",y-des)
desconto()